### ISSUE TITLE
PHP warnings in the quiz form

### ISSUE THREAD

**Issue by Willem3141** [graphchecker/moodle-qtype_graphchecker#38]
In GitLab by @Willem3141 on Feb 26, 2020, 11:09

```
Notice: Undefined property: qtype_graphchecker_question::$globalextra in /home/willem/Git/moodle/question/type/graphchecker/question.php on line 275

Notice: Undefined property: qtype_graphchecker_question::$attachments in /home/willem/Git/moodle/question/type/graphchecker/renderer.php on line 112

Notice: Undefined property: qtype_graphchecker_question::$showsource in /home/willem/Git/moodle/question/type/graphchecker/renderer.php on line 177
```

**Comment by Willem3141** [comment]
In GitLab by @Willem3141 on Feb 26, 2020, 12:08

closed via commit 7990d74edba885d914b95cabba47d993809853df

 *By Willem Sonke on 2020-02-26T11:08:07 (imported from GitLab project)*

**Comment by Willem3141** [comment]
In GitLab by @Willem3141 on Feb 26, 2020, 11:10

changed the description

 *By Willem Sonke on 2020-02-26T10:10:00 (imported from GitLab project)*
